utils/testsuite/testsuite.py

```diff
@@ -920,6 +925,13 @@ def get_arg_parser():
         action="store_true",
         help="Force lazy evaluation",
     )
+    parser.add_argument(
+        "--test-intl",
+        dest="test_intl",
+        default=False,
+        action="store_true",
+        help="Also test skiplisted INTL tests",
```

Comment:
I think this help message could be a little confusing. 

```suggestion
        help="Run supported Intl tests.",
```